src/core/database/errors.ts

```diff
@@ -55,6 +55,27 @@ export class ServiceUnavailableError extends Neo4jError {
   }
 }
 
+export class SessionExpiredError extends Neo4jError {
+  static readonly code = 'ServiceUnavailable' as const;
```

Comment:
```suggestion
  static readonly code = 'SessionExpired' as const;
```
Also need to call `fromNeo` in the `cast` function below.